examples/pytorch/pyg/BUILD

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+load("@rules_python//python:defs.bzl", "py_test", "py_library")
+load("@pip_deps//:requirements.bzl", "requirement")
+
+py_library(
```

Comment:
1. it should be a binary, so you can run it as an example.
2. Rename train.py to main.py. This is a Bazel convention I think.